peephole_benchmark_driver.py

```diff
@@ -0,0 +1,124 @@
+import subprocess
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+def do(command):
+	subprocess.call(command, shell=True)
+
+
+def run_one_circuit(timings, core_PL_timings, num_of_iters):
+	do(f"python3 my_toy_circuit.py {num_of_iters}")
+
+	with open('my_toy_circuit.yml', 'r') as f:
```

Comment:
Oh no, you can use a yaml package to parse it for you and access the elements like dictionaries, lists, etc :)